extension/intents/notes/notes.toml

```diff
@@ -16,6 +16,15 @@ match = """
 [[notes.addLink.example]]
 phrase = "Make note of this page"
 
+[notes.paste]
+description = "Focus the tab where you want to paste"
+match = """
+  paste (on | in | into |) note{s} (for me |)
+"""
+
+[[notes.paste.example]]
+phrase = "Paste everything present on the clipboard to notes"
```

Comment:
The example here should be something that is matched by the phrase above. So `Paste into notes` would be a good example.

`npm run jest` will also test examples, and should be giving you an error with this code (since the example isn't actually matched by the match phrase).